recipe/meta.yaml

```diff
@@ -5,10 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  fn: notebook-{{ version }}.tar.gz
+  fn: {{ version }}.tar.gz
```

Comment:
Good to know! I had issues with that before. (And now I have a few feedstocks to fix :smile_cat: )